### PR TITLE
Harfbuzz => 10.0.1 + downstream updates

### PR DESCRIPTION
**Pull request by satmandu**
- Adds `ConvenienceFunctions.libtoolize()`
##
Builds:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l`
##
### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/satmandu/chromebrew.git CREW_BRANCH=harfbuzz crew update \
&& yes | crew upgrade
```

### CODE REVIEW

File: packages/freetype.rb [hunk 1]
```diff
@@ -34,63 +34,5 @@ class Freetype < Meson
 
   def self.install
     system "DESTDIR=#{CREW_DEST_DIR} samu -C builddir install"
```
Comment (uberhacker) [inline packages/freetype.rb:36]: Can this whole section be removed too?

Comment (satmandu) [reply packages/freetype.rb]: Done.

File: packages/qt5_base.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@ class Qt5_base < Package
   version 'kde-5.15.14-9f9a56d-icu75.1'
   license 'GPL-2'
   compatibility 'x86_64 aarch64 armv7l'
-  min_glibc '2.37'
+  # min_glibc '2.37'
```
Comment (uberhacker) [inline packages/qt5_base.rb:9]: No rebuild necessary to make less than 2.37 compatible?

Comment (satmandu) [reply packages/qt5_base.rb:9]: I'm in the middle of an updated rebuild for it right now... But this is a dependency for something  `pangomm_2_48` needs but doesn't link to.

Comment (uberhacker) [reply packages/qt5_base.rb:9]: Should this PR be moved to draft then?

**Review by uberhacker — APPROVED** [review]

**Pull request merged**